src/shared/components/card-2/card-2.css

```diff
@@ -0,0 +1,102 @@
+/**
+ * Block
+ */
+
+.hw-card-2 {
+  position: relative;
+  background: var(--hw-color-white);
+  transition: transform 0.3s;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: var(--hw-spacing);
+
+
+  &::after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    transition: opacity 1s cubic-bezier(0.165, 0.84, 0.44, 1);
+    background: var(--hw-color-opaque-black-7);
+    content: '';
+    opacity: 0;
+  }
+
+  &:hover {
+    &::after {
+      opacity: 1;
+    }
+  }
+  &:hover .hw-card-2__body {
+    background-color: var(--hw-color-gray-lighter);
+  }
+
+  /**
+   * Elements
+   */
+
+  &__media {
+    overflow: hidden;
+    flex-grow: 1;
+  }
+
+  &__media-inner{
+      min-height: 100%;
+      min-width: 100%;
+      object-fit: cover;
+  }
+
+  &__body {
+    background-color: var(--hw-color-gray-lightest);
+    z-index: 1;
+    padding: var(--hw-spacing);
+    transition: all 0.1s ease;
+    border: 3px solid #f2f2f2;
+
+  }
+  &__label {
+    color: var(--hw-color-gray);
+    @extend .hw-text-uppercase;
+    @extend .hw-text-smaller;
+  }
+
+  &__title {
+    margin-bottom: var(--hw-spacing-smallest);
+    font-size: var(--hw-font-size-h4);
+  }
+
+  &__text {
+    display: inline-block;
+    min-height: 60px;
+    margin-top: var(--hw-spacing-smaller);
+    font-size: var(--hw-font-size-small);
+
+    @extend .hw-text-less-line-height;
+  }
+
+  @media(--max-medium){
+    flex-direction: row;
+    min-height: 150px;
+
+    &__media {
+      width: 60%;
+      min-width: 120px;
+    }
+    &:hover .hw-card-2__body {
+      border: 3px solid #C1C1C1;
+      border-left: 3px solid #E6E6E6;
+    }
+  }
+
+  @media(--large){
+    &__media {
+      max-height: 200px;
```

Comment:
guess this max-height should be replaced with height and placed under .hw-card-2__media-inner class.
also in design, the card has two variants ( one with a bigger image size in desktop view ). a separate class should handle it.